_data/projects/wospm-checker.yml

```diff
@@ -0,0 +1,12 @@
+name: WOSPM Checker
+desc: A checker for open source project owners to measure their open source project if welcoming or not
+site: https://wospm.github.io/checker/
+tags:
+  - php
+  - open-source
+  - oss
+  - metrics
+  - measure
+upforgrabs:
+  name: help wanted
+  link: hhttps://github.com/WOSPM/checker/labels/help%20wanted
```

Comment:
@umutphp This isn't a valid URL. Please remove extra `h` from the beginning.

Reply:
Sorry for that. Fixed :)